### ISSUE TITLE
`encryption` is incompatible with `files_primary_s3` and will result in corrupt files, but the docs don't mention it, and nothing prevents the admin from enabling both at once

### ISSUE THREAD

**Issue by posita** [owncloud/core#39263]
See #39212 for history, esp. [@JammingBen's comment](/owncloud/core/issues/39212#issuecomment-919842555).

**Comment by ho4ho** [comment]
Doc issue by: https://github.com/owncloud/docs/issues/4033

**Comment by JammingBen** [comment]
The above mentioned doc issue should at least clearify this in the docs again. Additionally we could enhance the WebUI (and the command?) with a warning.

**Comment by JammingBen** [comment]
> Additionally we could enhance the WebUI (and the command?) with a warning.

see https://github.com/owncloud/files_primary_s3/pull/487

**Comment by jnweiger** [comment]
## Not confirmed during QA

Seen with 
```
OC_Version = array(10,8,1,0);
$OC_VersionString = '10.8.1 prealpha';
$OC_Channel = 'daily';
$OC_Edition = 'community';
$OC_VersionCanBeUpgradedFrom = [[8,2,11],[9,0,9],[9,1]];
$OC_Build = '2021-11-17T22:03:13+00:00 803ffff1b2e22d11f40d953f36a3721a6da0779c';
```
files_primary_s3 1.1.3-rc1
encryption 1.5.1

* configure primary storage on s3 using a localhost scality server 
* `occ app:enable encryption` succeeds. Okayish.
* `occ encryption:enable` prints the expected error message: "Storage encryption is not compatible with S3 Object Storage." OK
* But the admin UI does not show any warnings. The admin can enable e.g. master key encryption. BAD
![image](https://user-images.githubusercontent.com/1108546/142498144-6f4cb146-e37d-4dac-a56d-0a964162adc7.png)
* Now `occ encryption:status` reports that the encryption is enabled. BAD
```
# occ encryption:status
  - enabled: true
  - defaultModule: OC_DEFAULT_MODULE
```

**Comment by jnweiger** [comment]
appinfo/info.xml has the new settings snippet.
appinfo has the the new register_command.php file
but there is no templates folder and no settings.php file.

**Comment by JammingBen** [comment]
> -> @JammingBen is that a make dist problem? e.g. owncloud/files_primary_s3@0684634

Ohh good point. The folders `js` and `css` are also missing I think.

**Comment by JammingBen** [comment]
@jnweiger I updated the release branch with the `js` and `css` folder. Could you please check again?

**Comment by jnweiger** [comment]
Built 1.1.3-rc2 from that, installed at https://oc1090beta1-primary-s3-113-rc2-20211119.jw-qa.owncloud.works/

Behaviour unchanged. Warning seen with occ, but admin UI still enables encryption on top of s3 without a word.
![image](https://user-images.githubusercontent.com/1108546/142616319-ee5d32b7-edd4-4ae7-9037-b4414d02593f.png)

**Comment by jnweiger** [comment]
Fixed in 1.1.3-rc3

![image](https://user-images.githubusercontent.com/1108546/142624718-8a0ba0b3-c123-480d-bea3-2779617735f9.png)
